app/views/pages/accessibility.html.haml

```diff
@@ -0,0 +1,68 @@
+- content_for :page_title_prefix  do
+  =t('static_pages.accessibility.page_title')
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %h1.govuk-heading-xl=t('static_pages.accessibility.page_title')
+
+    %h2.govuk-heading-l=t('static_pages.accessibility.title')
+    %p.govuk-body
+      =t('static_pages.accessibility.mission.opening_text')
+      %ul.govuk-list.govuk-list--bullet
+        -t('static_pages.accessibility.mission.list').each do |li|
+          %li=li.html_safe
```

Comment:
I'm not sure I'm a fan of this internationalisation, it feels really painful to have this separation.  What value does using it provide I wonder 🤔 .